src/test/java/org/folio/circulation/domain/ItemPermanentLocationTest.java

```diff
@@ -8,42 +8,40 @@
 
 import org.junit.jupiter.api.Test;
 
-import api.support.builders.HoldingBuilder;
 import api.support.builders.ItemBuilder;
 import lombok.val;
 
 class ItemPermanentLocationTest {
-
   @Test
   void itemLocationTakesPriorityOverHoldings() {
     val itemLocation = UUID.randomUUID();
-    val holdingsLocation = UUID.randomUUID();
+    val holdingsLocation = UUID.randomUUID().toString();
```

Comment:
Why does this need to be toString()?
Line 17 (`val itemLocation = ...`) appears to be doing the same thing as 18 but without the toString().

Reply:
Doesn't the holdings class require that this value be a string when used in the constructor?  Or are you really trying to ask "Why doesn't the holdings class store this as a UUID rather than a string?"

Reply:
Ok, so the `val` threw me off.
I was seeing line 18 and line 17 being the same but one used as a UUID and another a string.
I guess I am used to literal types more than `val` types.